packages/workflow/src/Interfaces.ts

```diff
@@ -733,22 +763,46 @@ export interface RequestHelperFunctions {
 	): Promise<any>;
 	requestWithAuthenticationPaginated(
 		this: IAllExecuteFunctions,
-		requestOptions: OptionsWithUri,
+		requestOptions: IRequestOptions,
```

Comment:
This method should really use `IHttpRequestOptions` 😕 but not easy/safe to change